lib/view/weeklyTaskView.dart

```diff
@@ -64,14 +61,14 @@ class _WeeklyTaskViewState extends State<WeeklyTaskView> {
   void loadPreviousWeek() async {
     await fetchData();
     setState(() {
-      today = today.subtract(Duration(days: 7));
+      today = today.subtract(const Duration(days: 7));
       generateScreen(today);
     });
   }
 
   void loadNextWeek() async {
     setState(() {
-      today = today.add(Duration(days: 7));
+      today = today.add(const Duration(days: 7));
```

Comment:
Im gonna change this later since Duration(days: i) accounts for daylight savings and such but we dont care and just want 7 days from now

Reply:
![image](https://github.com/Scrumbags115/plannertarium/assets/122345910/8a82a7dd-ee48-414e-a9b8-be1a264f0c58)
`add` and `subtract` seem inconsistent with its behavior due to daylight savings and whatnot, we could switch to using what this website recommends?
![image](https://github.com/Scrumbags115/plannertarium/assets/122345910/051e6026-12a1-4f88-9672-74c5dc3cadaa)

Reply:
I have this functionality in getDateOnly(offset: 7) where it does that.

Reply:
alright, i'll leave it to you then